### DISCUSSION TITLE
Possible delay for Part B - Team leader local repo push to remote including the new Dev branch

### DISCUSSION THREAD

**Discussion by MichaelKelly9** [MichaelKelly9/Tafe24S1T104_Ext#2] [General]
Hi Zihao,

Apologies for not finishing Part B yet. I had planned to finalize and push my local repo including the dev branch with Main menu coded into the remote repo today. However I have got a power outage in my area so may be unable to complete that push today. 

I will try to get to a library for power/internet to complete Part B today but that is dependent on whether the local daycare closes due to the power outage and I would need to then look after my daughter.

Worst Case I will get it finalized tomorrow morning. I will give you an update as soon as it has been pushed into GitHub

**Comment by MichaelKelly9** [comment]
Hi @ZIHAOJIA 

I have successfully completed the main menu addition and pushed my local repo into GitHub today.
Feel free to proceed with Part C (individual work). I have emailed KT to double check about your public key as I do not see anywhere to add it into the GitHub Repo.

As a reminder we will be Branching off using these names for the new features, please advise which branch you would prefer to work on as I have forgotten what was decided in the Teams meeting.

feature1_currency_con
feature2_mortgage_calc

Thanks
Michael

**Reply by ZIHAOJIA** [reply]
I want to duoble check,which is the mortgage_calc, it it is feature2,i will do feature2_mortgage_calc.

**Reply by MichaelKelly9** [reply]
Hi @ZIHAOJIA ,
Yes feature 2 is the mortgage calculator. I will leave that up to you and I will begin work on feature 1 the Currency Converter

**Reply by ZIHAOJIA** [reply]
all right
